README.md

```diff
@@ -19,11 +19,11 @@ Supported versions:
 
 Use the `go` command:
 
-	$ go get github.com/satori/go.uuid
+	$ go get github.com/gofrs/uuid
 
 ## Requirements
 
-UUID package requires Go >= 1.2.
+UUID package requires Go >= 1.3.
```

Comment:
I don't think this is technically true. We stopped testing 1.2 because of our wanting to continue using coveralls.io, I'm not sure anything in the library itself is broken with `< Go 1.2`. Admittedly, because there is no context provided in da9da744cf7ea776632325cce5335dc49481529f someone may not know why we dropped Go 1.2 support.

We could maybe rephrase this to be more correct, in that this library is only **_tested_** against Go 1.3+.